lambda/custom-collection/scripts/whitelistAndMint.js

```diff
@@ -0,0 +1,63 @@
+import env from "dotenv";
+import { invoke } from "../../lambdaUtils.mjs";
+import { solidityKeccak256 } from "ethers/lib/utils.js";
+import {
+  waitUntilTransactionMined,
+  getUserWallet,
+  sleep,
+  checkAPIKey,
+} from "../../../helpers/utils.mjs";
+
+async function main() {
+  env.config();
+
+  // Checks if the API_KEY is set in the .env file.
+  checkAPIKey();
+
+  // Gets the user Address to whitelist in NFT collection contract.
+  const userAddress = await getUserWallet(process.env.USER_EMAIL);
+
+  /* *************************************************************** Whitelist User *************************************************************** */
+  console.log(
+    "***************************************************** Whitelist User *****************************************************\n"
```

Comment:
reduce length of these lines. Lines should be <100 chars in general

Reply:
done